applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -137,21 +142,52 @@ For details, see the `Testing`_ section.
 User interface
```

Comment:
```suggestion
.. _matter_bridge_app_ui:

User interface
```

Reply:
The user interface section was unnecessarily removed, please add below the .. _matter_bridge_app_ui: part:
``User interface
**************``